lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -68,13 +67,13 @@ def self.details
       end
 
       def self.available_options
-        platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
+        @platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
```

Comment:
Here you are relying on the fact that `available_options` is called before `run`. While this may be the case now, the core Fastlane code that calls these methods could change at any time. Also if someone wanted to modify this class later to use `@platform`, they would have to recognize this and figure out whether or not it would work for their use case. In general, requiring methods to be called in a particular order should be almost always avoided.

A more appropriate pattern here would just be to add a `platform` method that simply returned `Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]`.

If that was an expensive operation, and it was not expected to change, you could memoize it (https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/):
```
def self.platform
  @platform ||= Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
end
```

But I doubt this is expensive. It probably just looks up a value in a hash. So a simple method would work.

Reply:
Sounds good I agree with this change.